config/application.rb

```diff
@@ -16,6 +16,7 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.exceptions_app = self.routes
```

Comment:
Nice one @essymo It's looks like this bit was missed when the pages were added in d2799fe25c7352c90b75611e69dcae8431118b9b